related_images.developer.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
I am supportive of this change as we could use a developer bundle to run e2e on hypershift.